src/app/utils.ts

```diff
@@ -347,6 +351,55 @@ export function detectBrowserLanguage(defaultLanguage: string = "en"): string {
   return defaultLanguage;
 }
 
+// Gets amount precision for each token traded on dexteronradix.
+// Note: precision for price is different.
+export function getPrecision(input: string): number {
+  return (
+    {
+      XRD: 2,
+      DEXTR: 2,
+      CASSIE: 0,
+      CAVIAR: 2,
+      DFP2: 2,
+      DGC: 0,
+      FLOOP: 5,
+      HUG: 0,
+      MNI: 0,
+      OCI: 2,
+      PLANET: 0,
+      RDK: 0,
+      WEFT: 2,
+      XRAW: 0,
+      XUSDC: 2,
+    }[input.toUpperCase()] || 2
+  );
+}
```

Comment:
We need to make sure we replace this with a dynamic lookup. 
I am not sure how this is used, but there is already a max decimals value for each token in a pair in the pairInfo. I dont think 2 is enough for most use cases.

Reply:
This is currently only used to display the precision in the "available: 1'000.00 XRD" label. 

But I agree we should figure out dynamic precision that automatically adapts when new tokens get added. I am tracking this issue here: https://github.com/DeXter-on-Radix/website/issues/371